frappe/patches/v12_0/set_datetime_on_reset_password_key.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import unicode_literals
+
+import frappe
+
+
+def execute():
+	frappe.db.sql("""
+UPDATE `tabUser`
+SET
+	expire_reset_password_key = getdate()
```

Comment:
@Thunderbottom `getdate()` is not compatible with postgres user `NOW()` instead

& 

fix code formatting